Servers/driver/autoDriver.driver.ts

```
@@ -684,72 +528,25 @@ export async function insertMockData() {
-      topics![23].id,
-      topics![24].id,
-      topics![25].id,
-    ).map((d) => subTopicGenerateValuesString(d as any));
+    const values = subTopicMockData(topics!.map(t => t.id)).map((d) => subTopicGenerateValuesString(d as any));
     insertString += values.join(",") + "RETURNING id;";
     subTopics = await insertData(insertString as string);
   }
 
   var {
     mockData: questionMockData,
     tableName,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'tableName'. Consider to delete it or rename it.

'tableName' is defined here:


(lint/suspicious/noRedeclare)